services/search/testing/test_search_engine.py

```diff
@@ -1,5 +1,70 @@
+import mock  # pylint: disable=import-error
```

Comment:
nit: if you do `from unitttest import mock` there is no need to disable the error

Reply:
Really weird since they should be the same according to https://pypi.org/project/mock/

Reply:
If you are using the `mock` from `pip` that is why you are getting an error, since it is an external library. the one I suggested is internal and that is why it doesn't require the error suppresion. Any reason why you are not using the integrated `unittest`?